### ISSUE TITLE
rafting not working properly (too small raft area)

### ISSUE THREAD

**Issue by goofyseeker311** [Ultimaker/Cura#15526]
### Application Version

5.3.1

### Platform

win11

### Printer

elegoo neptune 3 pro

### Reproduction steps

inverted cone (sphere/bowl) will have a very small raft area which detaches easily,
if "raft extra margin" is increased it complains its outside the print area.
what is the proper setting for making the raft area larger, should apply locally to the raft area and not globally.

### Actual results

did not get larger raft area, but complaints is outside the print area, using expert settings for adhesion with raft

### Expected results

raft extra margin should apply only locally, ie to the small raft area in the center of the build plate, not like the raft is near the edges. A print that is bed size wide on top, but 20% at the bottom where the adhesion raft is.

### Checklist of files to include

- [ ] Log file
- [X] Project file

### Additional information & file uploads

[bowl.3mf.zip](https://github.com/Ultimaker/Cura/files/11477570/bowl.3mf.zip)

**Comment by GregValiant** [comment]
Thanks for the report and the file.
I can duplicate this.  The raft is tiny because the "Raft Extra Margin" is constrained by the size of the model versus the size of the bed.
The Cura team will take a look.
This is as big as I can get the raft in 5.3.1 and in 4.13.1.
![image](https://github.com/Ultimaker/Cura/assets/64202104/fac1098d-69ae-4865-8597-81af4f1267f9)

**Comment by goofyseeker311** [comment]
Also,
when slicing a wavy small feature grill grid (sub 1cm pattern) then somehow
the slicer loses parts or the wall pattern, like detail.

**Comment by GregValiant** [comment]
That sounds like a different problem.  If Cura is dropping features it is often because they are too fine for the nozzle and/or line width you have set.  You can check by dropping your nozzle size, re-slicing, and checking the preview.
A second reason is that there are errors in the model.

**Comment by goofyseeker311** [comment]
yes its a different problem, related to slicing behaviour. most probably there will not be a wall (even one line thick) if its too narrow. there should be some kind of option to ensure no walls will be culled out, ie, they will be preserved, even if below the minimum wall /nozzle thickness values. I had to scale up the model several times to get the walls to appear properly. and obviously cant readily downsize the physical nozzle, from the usual 1.75mm nozzle in the printer.

just commenting on the default slicing strategy/behaviour.

**Comment by goofyseeker311** [comment]
well, first directive contradicts the second, by not making the walls, as in the model, but altering the model. just saying. :)

**Comment by GregValiant** [comment]
Here is a knife edge model.  At the top is a "didn't slice because it's too narrow" area.
![image](https://github.com/Ultimaker/Cura/assets/64202104/e8ea87bc-6f9e-49e2-b1f0-219b08310e47)

Along with features that are just too thin (your walls), this happens with spires, wedges, or any model where the cross section of a feature falls below "Minimum Line Width".  The software would need to make a decision (adlib) on how to treat the top of this model, or the missing wall in your model.  It would have to over-ride the user setting of Minimum Line Width.  That would seem to me to require some sort of "AI" to make a decision based on the intent of the designer of the model, and that would seem to go beyond "Slicing Software".
